datafusion/physical-plan/src/joins/nested_loop_join.rs

```diff
@@ -238,6 +238,19 @@ impl NestedLoopJoinExec {
 
         PlanProperties::new(eq_properties, output_partitioning, mode)
     }
+
+    fn maintains_input_order(join_type: JoinType) -> Vec<bool> {
```

Comment:
I feel it would be great to have some doc on what this method does and return. Specifically what false/true means, what is the benefit of preserving the order

Reply:
Addressed in 09a6ceee435e1237a361b87d059b34f4fac03385